src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/view.html.twig

```diff
@@ -115,3 +115,9 @@
 
   <script src="{{ asset('themes/new-theme/public/order_view.bundle.js') }}"></script>
 {% endblock %}
+
+{% set js_translatable = {
+  "Items successfully added": 'Succesfully added [1] item(s).'|trans({}, 'Admin.Notifications.Success'),
```

Comment:
Wording question for @Julievrz or @LouiseBonnard First foes this kind of wording seem ok to you? Regarding the way to handle plural
Second, should we use a generic wording (using "items") or a specific one for this page (using "products")

Reply:
@jolelievre I am not a fan of this way to handle the plural form because it can be very difficult to translate for languages who have a completely different word for plural form (e.g.: Polish). 

I think the best solution would be to have 2 strings. One in singular "The product was successfully added/removed."  and one in plural for bulk actions ("Products were successfully added/removed") 

If we are talking about products only here then you are right, using the word "product" makes more sense, thank you for noticing it. 😊

Reply:
@sowbiba Do you think it would be possible to have 2 different wordings as explained above? 😊